tethys_portal/settings.py

```diff
@@ -137,7 +134,7 @@
     'handlers': ['console_verbose'],
     'level': 'INFO',
 })
-LOGGERS.setdefault('tethys.apps', {
+LOGGERS.setdefault('tethysapps', {
```

Comment:
```suggestion
LOGGERS.setdefault('tethysapp', {
```

this needs to be the same name as the namespace so that apps can just do:

```python
logger = logging.getLogger(__name__)
```